dev.ensoft.ecode.yml

```diff
@@ -0,0 +1,78 @@
+app-id: dev.ensoft.ecode
```

Comment:
`ensoft.dev` is mentionned nowhere and the website is broken. Is this really upstreams?

Reply:
so why isn't the homepage on the appstream file set to it?

Reply:
Hi! Author of ecode here. There's no landing page for the project at the moment. But I'm using that domain to host some documentation and the web version.
You can see the domain is mentioned in: https://github.com/SpartanJ/ecode#live-demo, the demo is located at: https://cdn.ensoft.dev/eepp-demos/demo-fs.html?run=ecode.js

Reply:
Give me some minutes and I'll register `ecode.ensoft.dev` and redirect to `https://github.com/SpartanJ/ecode`. I would love to have `ecode.dev` but it's registered, I'll take a look for a domain for the project.

Reply:
it's not blocker. Just that there are so many people you decided using random domain that I have to double check.

Reply:
`ecode.ensoft.dev` is already working